packages/react-ui/src/app/builder/piece-properties/auto-properties-form.tsx

```diff
@@ -293,6 +293,12 @@ const selectFormComponentForProperty = ({
     case PropertyType.BASIC_AUTH:
     case PropertyType.OAUTH2:
       return <></>;
+    case PropertyType.CUSTOM:
+      return <CustomProperty
+        code={property.code}
+        value={field.value}
+        onChange={field.onChange}
```

Comment:
It would be nice to have a `disabled` prop as well so that we can disable the button in the workflow run view.